### ISSUE TITLE
configservice_lenovo : Segfault on Debian/Sid

### ISSUE THREAD

**Issue by phaoost** [lenovo/lenovo-wwan-unlock#13]
Hello.

I am getting the following on the recent Debian/sid:
```
~# /opt/fcc_lenovo/configservice_lenovo 
Lenovo Config Service Version 1.0
Error: signal 11:
/usr/lib/libconfigservice350.so(_Z14debugBackTraceiP9siginfo_tPv+0x30)[0x7faa343b2a60]
/lib/x86_64-linux-gnu/libc.so.6(+0x3d580)[0x7faa34658580]
/lib/x86_64-linux-gnu/libstdc++.so.6(_ZNKSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE7compareEPKc+0x14)[0x7faa33f537e4]
/usr/lib/libconfigservice350.so(_Z19fm350_configservicev+0x1e1)[0x7faa343b63d1]
/opt/fcc_lenovo/configservice_lenovo(+0x352f)[0x55cc2b97152f]
/opt/fcc_lenovo/configservice_lenovo(+0x37b2)[0x55cc2b9717b2]
/lib/x86_64-linux-gnu/libc.so.6(+0x27c8a)[0x7faa34642c8a]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x85)[0x7faa34642d45]
/opt/fcc_lenovo/configservice_lenovo(+0x25e5)[0x55cc2b9705e5]
```

**Comment by nitinexclusively** [comment]
Hello @phaoost Thanks for this issue. This is expected to be fixed in next release around end of June . I will let you know , once trial version is ready for release . Thanks

**Comment by nitinexclusively** [comment]
@phaoost Sorry to keep you waiting . Can you please retry by taking latest package from main branch . Thank you !

**Comment by nitinexclusively** [comment]
closing this ticket . Please open new issue , if there is any other issue . Thanks
